### DISCUSSION TITLE
umm Lại điểm danh nhé kakakakakakakakkakakakakakkakaa

### DISCUSSION THREAD

**Discussion by dangxuantruong10a13** [dangxuantruong10a13/nhom9_10a13#24] [Announcements]
hi
Chỗ này mình sẽ thảo luận bài mà thầy giao

**Comment by tranphamkhuongduy10a13pn** [comment]
hello

**Comment by dangxuantruong10a13** [comment]
umm, bên phần code thì các pặn up file mà mình làm lên để lưu lại nha ae hoặc là để mn lấy bài để tổng hợp lại
